sdk/mixedreality/Azure.MixedReality.RemoteRendering/README.md

```diff
@@ -0,0 +1,386 @@
+# Azure Remote Rendering client library for .NET
+
+Azure Remote Rendering (ARR) is a service that enables you to render high-quality, interactive 3D content in the cloud and stream it in real time to devices, such as the HoloLens 2.
+
+This SDK offers functionality to convert assets to the format expected by the runtime, and also to manage
+the lifetime of remote rendering sessions.
+
+> [!Note] Once a session is running, a client application will connect to it using one of the "runtime SDKs".
+> These SDKs are designed to best support the needs of an interactive application doing 3d rendering.
+> They are available in ([.net](https://docs.microsoft.com/dotnet/api/microsoft.azure.remoterendering)
+> or ([C++](https://docs.microsoft.com/cpp/api/remote-rendering/)).
+
+[Product documentation](https://docs.microsoft.com/azure/remote-rendering/)
```

Comment:
nit: we typically don't include a TOC in the README. I don't think it looks bad but is inconsistent with other readmes.